source/puppet/5.5/modules_documentation.markdown

```diff
@@ -194,116 +194,103 @@ To transition your module to the `REFERENCE.md` model, add Strings comments to y
 {:.section}
 ### Manually writing reference information
 
-If you aren't using Puppet Strings yet to generate your Reference documentation, you can manually create a `REFERENCE.md` file listing each of your classes, defined types, resource types and providers, functions, and facts, along with any parameters. If your module contains type aliases or facts, which Strings doesn't support yet, document them in a short, supplementary Reference section.
+If you're not yet ready to use Puppet Strings to generate your reference docs, you can manually create a `REFERENCE.md` file listing all of your module's classes, defined types, functions, tasks, task plans, and resource types and providers. This is useful if you are in the process of moving your reference documentation into code comments, or if your module contains resources that aren't yet supported in Strings.
 
-To manually document reference information, start your reference information with a small table of contents that first lists the classes, defined types, and resource types of your module. If your module contains both public and private classes or defined types, list the public and the private separately. Include a brief description of what these items do in your module.
+If your module contains a `REFERENCE.md` file, the Forge displays the contents in a Reference tab on the module's detail page, whether you created the file manually or with Strings. Write the `REFERENCE.md` file in Markdown, and keep it in the root directory of your module.
 
-```
-## Reference
-
-### Classes
-
-#### Public classes
-
-*[`pet::cat`](#petcat): Installs and configures a cat in your environment.
+If you already have a README with an accurate reference section, you can remove that section from your README and paste it into a `REFERENCE.md` file. The formatting isn't vastly different.
 
-#### Private classes
+> **Important**: If you create a `REFERENCE.md` manually, and then generate Markdown with Puppet Strings, it will overwrite your `REFERENCE.md` unless you give the files different name. The module detail page on the Forge displays only the `README.md` and `REFERENCE.md` files.
```

Comment:
"...it will overwrite your `REFERENCE.md` unless you give the files different name*s*."?

Reply:
Oops! I kept going back and forth, is what happened.